src/main/java/org/javamodularity/moduleplugin/tasks/TestTask.java

```diff
@@ -40,14 +40,15 @@ public void configureTestJava() {
                 .ifPresent(this::configureTestJava);
     }
 
+    @SuppressWarnings("Convert2Lambda")
```

Comment:
Added this to stop the use of code that can be converted to lambda, but which shouldn't be, showing up as a warning when compiling.